libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/arrow/api/MapboxRouteArrowView.kt

```diff
@@ -176,7 +176,11 @@ class MapboxRouteArrowView(private val options: RouteArrowOptions) {
     }
 
     private fun updateLayerVisibility(style: Style, layerId: String, visibility: Visibility) {
-        style.getLayer(layerId)?.visibility(visibility)
+        style.getLayer(layerId)?.apply {
+            if (this.visibility != visibility) {
```

Comment:
i was wondering why not put this into the map style layer and then found it would have to be done all over the place

![Screen Shot 2022-09-09 at 1 16 49 PM](https://user-images.githubusercontent.com/3021882/189436083-5c9b13c9-9ab5-472a-aa26-a2e03fde5acf.png)